test_benchmarks.py

```
@@ -0,0 +1,37 @@
+import pytest
+from main import MarkdownScraper
+
+@pytest.mark.benchmark(group="scrape_website")
+def test_scrape_website_benchmark(benchmark):
```

Ruff (0.8.2)

Missing return type annotation for public function `test_scrape_website_benchmark`

Add return type annotation: `None`

(ANN201)